security/acme-client/src/opnsense/mvc/app/library/OPNsense/AcmeClient/LeValidation/DnsMijnHost.php

```diff
@@ -0,0 +1,44 @@
+<?php
```

Comment:
The filename needs to be changed: `DnsMijnHost.php -> DnsMijnhost.php` (lower case `h`).